ArcGISRuntimeSDKQt_CppSamples/Scenes/OrbitCameraAroundObject/OrbitCameraAroundObject.qml

```diff
@@ -0,0 +1,210 @@
+                y: cameraHeadingSlider.topPadding + cameraHeadingSlider.availableHeight / 2 - height / 2
+                implicitWidth: 30
+                implicitHeight: 30
+
+                Text {
+                    id: headingValue
+
+                    anchors {
+                        horizontalCenter: parent.horizontalCenter
+                        verticalCenter: parent.verticalCenter
+                    }
+
+                    text: Math.round(cameraHeadingSlider.value)
+                    color: "black"
+                }
+            }
+        }
+
+        Text {
+            id: cameraHeadingLabel
+
+            anchors {
+                horizontalCenter: parent.horizontalCenter
+                bottom: cameraHeadingSlider.top
+            }
+
+            text: "Camera Heading"
+            color: "white"
+        }
+    }
+
+    //Plane pitch slider, placed in the top-right of the screen
+    Rectangle {
+        anchors {
+            top: parent.top
+            bottom: parent.verticalCenter
+            right: parent.right
+            margins: 30
+        }
+
+        width: childrenRect.width
+        color: uiBackgroundCol
+
+        ColumnLayout {
+            anchors {
+                top: parent.top
+                bottom: parent.bottom
+            }
+            spacing: 5
+
+            Text {
+                id: planePitchLabel
+                text: "Plane Pitch"
+                color: "white"
+
+            }
+
+            Slider {
+                id: planePitchSlider
+
+                Layout.fillHeight: true
+
+                value: 0
+                from: 90
+                to: -90
+                orientation: Qt.Vertical
+
+                //Custom slider handle that displays the current value
+                handle: Rectangle {
+                    x: planePitchSlider.leftPadding + planePitchSlider.availableWidth / 2 - width / 2
+                    y: planePitchSlider.topPadding + planePitchSlider.visualPosition * (planePitchSlider.availableHeight - height)
+
+                    implicitWidth: 30
+                    implicitHeight: 30
+
+                    Text {
+                        id: pitchValue
+
+                        anchors {
+                            horizontalCenter: parent.horizontalCenter
+                            verticalCenter: parent.verticalCenter
+                        }
+
+                        text: Math.round(planePitchSlider.value)
+                        color: "black"
+                    }
+                }
+            }
+        }
+    }
+
+    //View change buttons / allow cam interaction checkbox placed in the top-left of the screen.
+    Rectangle{
+
+        anchors {
+            left: parent.left
+            top: parent.top
+        }
+
+        height: childrenRect.height
+        width: childrenRect.width //Add a little bit of manual padding to comfortably fit the checkbox text
+        color: uiBackgroundCol
+
+        Column {
```

Comment:
If you use a `ColumnLayout` as opposed to a `Column` you don't need to next an additional `Row`. Just use `Layout.columnSpan: 2` on the buttons.